Semantic Scholar.js

```diff
@@ -680,16 +639,19 @@ var testCases = [
 						"creatorType": "author"
 					},
 					{
-						"firstName": "Peter",
+						"firstName": "Peter E.",
 						"lastName": "Clark",
 						"creatorType": "author"
 					}
 				],
 				"date": "2018",
-				"abstractNote": "We present a new dataset and models for comprehending paragraphs about processes (e.g., photosynthesis), an important genre of text describing a dynamic world. The new dataset, ProPara, is the first to contain natural (rather than machine-generated) text about a changing world along with a full annotation of entity states (location and existence) during those changes (81k datapoints). The end-task, tracking the location and existence of entities through the text, is challenging because the causal effects of actions are often implicit and need to be inferred. We find that previous models that have worked well on synthetic data achieve only mediocre performance on ProPara, and introduce two new neural models that exploit alternative mechanisms for state prediction, in particular using LSTM input encoding and span prediction. The new models improve accuracy by up to 19%. The dataset and models are available to the community at http://data.allenai.org/propara.",
+				"DOI": "10.18653/v1/N18-1144",
+				"abstractNote": "We present a new dataset and models for comprehending paragraphs about processes (e.g., photosynthesis), an important genre of text describing a dynamic world. The new dataset, ProPara, is the first to contain natural (rather than machine-generated) text about a changing world along with a full annotation of entity states (location and existence) during those changes (81k datapoints). The end-task, tracking the location and existence of entities through the text, is challenging because the causal effects of actions are often implicit and need to be inferred. We find that previous models that have worked well on synthetic data achieve only mediocre performance on ProPara, and introduce two new neural models that exploit alternative mechanisms for state prediction, in particular using LSTM input encoding and span prediction. The new models improve accuracy by up to 19%. The dataset and models are available to the community at this http URL",
+				"extra": "arXiv: 1805.06975",
 				"itemID": "Dalvi2018TrackingSC",
 				"libraryCatalog": "Semantic Scholar",
 				"proceedingsTitle": "NAACL-HLT",
+				"publicationTitle": "NAACL-HLT",
```

Comment:
Are both fields here come from Scaffold?

This looks strange as `proceedingsTitle` is only some sort of alias to `publicationTitle`...

Reply:
Scaffold doesn't work well with updating tests for some reason -- the [element](https://github.com/zotero/translators/pull/2103/files#diff-ebdd943ca6e2a8dcc2fe2810642d2cc9L53) used to determine type in `detectWeb` isn't found for some reason, even with `defer: true`. This is despite it appearing when I wget or curl a page. Not sure why this is, but the tests run/pass as usual.